boards/xtensa/intel_adsp_cavs25/intel_adsp_cavs25_2.conf

```diff
@@ -0,0 +1,2 @@
+# board revision 2 = only 2 DSP cores available
+CONFIG_MP_NUM_CPUS=2
```

Comment:
The missing new line could cause issues when concatenating or other?

Reply:
Let me fix that.